sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- `tracingPolicy` will no longer propagate tracing errors to the caller, instead logging and continuing. [PR #16916](https://github.com/Azure/azure-sdk-for-js/pull/16916)
```

Comment:
this one could be updated similar to the other?
```suggestion
- `tracingPolicy` will no longer propagate tracing errors to the caller, and such errors will be logged instead and the operation does not get interrupted. [PR #16916](https://github.com/Azure/azure-sdk-for-js/pull/16916)
```

Reply:
🤦‍♂️ thank you!